recipes/simplekml/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.3.0" %}
+
+package:
+    name: simplekml
+    version: {{ version }}
+
+source:
+    fn: simplekml-{{ version }}.zip
+    url: https://pypi.python.org/packages/source/s/simplekml/simplekml-{{ version }}.zip
+    md5: a9f133bde1c0edff6166739faccc11a1
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+
+test:
+    imports:
+        - simplekml
+
+about:
+    home: http://code.google.com/p/simplekml/
+    license: GPL-3.0
```

Comment:
Believe this LGPL3. Also it has moved ( https://bitbucket.org/KyleLancaster/simplekml ).